src/schema/axis.schema.ts

```diff
@@ -1,3 +1,18 @@
+export interface Axis {
```

Comment:
`interface-name enforces the rule that interface names must begin with a capital 'I'`

https://github.com/palantir/tslint